Lib/Inc/CCmdBuffer.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file CCmdBuffer.h
+ */
+
+/*
+ *  Created on: Mar 28, 2022
+ *      Author: niuslar
+ */
+
+#ifndef CCMDBUFFER_H_
+#define CCMDBUFFER_H_
+
+#include <string>
+#include "main.h"
+
+#define BUF_SIZE 60
+
+class CCmdBuffer
```

Comment:
@Niuslar Using C++ templates is quite common for circular buffers.  The buffer becomes data type agnostic.

Reply:
Thanks, I haven't worked much with templates yet. I'll have a look at this.